requirements-mpi.txt

```diff
@@ -1 +1,2 @@
-mpi4py
+mpi4py<4.0
```

Comment:
NFR: this was added because soon mpi4py will go to version 4 after long time of minor upgrades

Reply:
Does dependabot actually understand this is a dependency file as well?